packages/analytics/metric-cards/src/components/display/MetricsCard.vue

```diff
@@ -0,0 +1,233 @@
+<template>
+  <div class="metricscard">
+    <div class="metricscard-title">
+      <span>{{ title }}</span>
+      <KTooltip
+        v-if="tooltip"
+        class="metricscard-tooltip"
+        :label="tooltip"
+        placement="right"
+      >
+        <KIcon
+          icon="info"
+          size="12"
+        />
+      </KTooltip>
+    </div>
+    <div
+      v-if="hasError"
+      class="metricscard-error"
+    >
+      <KIcon
+        icon="warning"
+        size="20"
+      />
+      <div>{{ errorMessage }}</div>
+    </div>
+    <div
+      v-else
+      class="metricscard-value"
+    >
+      <div :style="`font-size:${metricFontSize}`">
+        {{ metricValue }}
+      </div>
+      <div
+        v-if="cardDisplayFull"
+        class="metricscard-value-trend "
+        :class="textColor(changePolarity)"
+      >
+        <KIcon
+          v-if="changePolarity !== 0"
+          :color="colorAttribute(changePolarity)"
+          :icon="icon"
+          size="18"
+        />
+        <div>
+          {{ metricChange }}
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { PropType } from 'vue'
+import { KIcon, KTooltip } from '@kong/kongponents'
+import { MetricCardSize } from '../../types'
```

Comment:
```suggestion
import type { MetricCardSize } from '../../types'
```

Reply:
Hmmm, if i make this change, then VS Code complains when I use it for a value comparison:

<img width="962" alt="Screen Shot 2023-02-24 at 10 15 27 AM" src="https://user-images.githubusercontent.com/103061463/221263539-2f19b274-a730-4dee-83eb-15a3104fe64a.png">

I do this a couple of times to determine which elements are shown or hidden, eg:
```
const cardDisplayFull = [MetricCardSize.Medium, MetricCardSize.Large].includes(props.cardSize)
```

Reply:
ah it's an enum; ignore.